sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -165,6 +165,8 @@ abstract class Optimizer(sessionCatalog: SessionCatalog)
     Batch("LocalRelation", fixedPoint,
       ConvertToLocalRelation,
       PropagateEmptyRelation) :+
+    Batch("Extract PythonUDF From JoinCondition", Once,
+        HandlePythonUDFInJoinCondition) :+
     // The following batch should be executed after batch "Join Reorder" and "LocalRelation".
```

Comment:
please update this comment too with the new rule

Reply:
Thanks, done in 98cd3cc.